mule-extensions-api-persistence/src/main/java/org/mule/runtime/extension/internal/persistence/ExtensionModelTypeAdapter.java

```diff
@@ -134,6 +136,10 @@ public ExtensionModel read(JsonReader in) throws IOException {
 
     Set<ObjectType> types = parseTypes(TYPES, json);
 
+    JsonArray notifications = json.get(NOTIFICATIONS).getAsJsonArray();
```

Comment:
to keep this compatible with 1.0.0 models, check if NOTIFICATIONS exists in the json, and if not present create an empty notifications array. Otherwise a 1.0 model that is loaded in 1.0 will throw an NPE